src/main/kotlin/racingcar/Game.kt

```diff
@@ -0,0 +1,23 @@
+package racingcar
+
+class Game(
+    carNumber: Int,
+    private val tryCount: Int,
+) {
+    private val cars: Array<Car> = Array(carNumber) { Car() }
+
+    fun run(): List<GameResult> {
+        val gameResult = mutableListOf<GameResult>()
+        for (stage in (0 until tryCount)) {
+            gameResult.add(runGameOneStage())
+        }
```

Comment:
repeat 함수를 활용해볼 수도 있을 것 같아요. 😉
```suggestion
        repeat (tryCount) {
            gameResult.add(runGameOneStage())
        }
```